test_syndiffix/onePrivMeasure.py

```diff
@@ -13,7 +13,7 @@
 '''
 
 
-def onePrivMeasure(jobNum=0, runsDir='runAb', measuresDir='measuresAb', resultsDir='resultsAbHalf1', controlDir='csvAbHalf2', force=False):
+def onePrivMeasure(jobNum=0, runsDir='runAb', measuresDir='measuresAb', resultsDir='resultsAb', controlDir='csvAb/test', force=False):
```

Comment:
Simply changing controlDir certainly does the trick. A cleaner way more in line with the new setup is to get rid of controlDir altogether, and instead of doing this:

```
        self.dfControl = pd.read_csv(controlFilePath, low_memory=False, skipinitialspace=True)
```

you do this:
```
        dfControl = pd.DataFrame(results['testTable'], columns=results['colNames'])
```

This is possible because we now keep the "test" data in the results json file itself.

I should have pointed this out earlier...

Reply:
ah, you did mention it, I forgot, will update